src/explicit.cc

```diff
@@ -136,6 +136,24 @@ namespace hpp {
     {
     }
 
+    Explicit::Explicit
+    (const LiegroupSpacePtr_t& configSpace,
```

Comment:
Any reason to keep the config space here ? It isn't used.

Reply:
I removed the useless parameter and pushed.
Thank you for your review.

Reply:
Gcc has been more careful than I.